app/src/components/field/TextField.tsx

```diff
@@ -25,4 +29,21 @@ function TextField(props: TextFieldProps, ref: Ref<HTMLDivElement>) {
 }
 
 const _TextField = forwardRef(TextField);
-export { _TextField as TextField };
+
+const NumberField = forwardRef(function NumberField(
```

Comment:
This is basically just a TextField as far as react aria is concerned but with extra props... not sure why they distinguish like this

Reply:
Minor nit: I think it would be easier to find if it was it's own file